### DISCUSSION TITLE
Debezium would like to join CommonHaus

### DISCUSSION THREAD

**Discussion by Naros** [commonhaus/foundation#210] [Joining Commonhaus]
### Project information

- **Project name**: Debezium
- **Project website**: https://debezium.io
- **Code repository**: https://github.com/debezium/debezium (main project repository)
- **License**: Apache 2.0

### Do you have authority to represent this project?

- [X] I own or maintain this project

### Describe your project

Debezium is an open-source distributed platform designed for Change Data Capture (CDC), providing applications with an industry-standard way to respond to real-time database changes. It simplifies the process of monitoring and capturing data changes across various databases by providing a robust and scalable solution to stream database change vectors as events. Debezium’s architecture is built on the concept of connectors that interact with and consume database transaction logs, ensuring that every insert, update, or delete operation is captured and published to an event stream, most often using Kafka. This empowers organizations to build event-driven architectures, facilitate microservice communications, and maintain data consistency across distributed systems.

Debezium’s importance lies in its ability to provide real-time insights and integration, which is critical in today’s hyper-focused, data-driven world. Businesses can enhance their decision-making processes, improve user experiences, and ensure data integrity by allowing applications and systems to react instantaneously to changes. Debezium’s integration with Apache Kafka, Quarkus, Strimzi, and Kubernetes emphasizes our open-source first principles, making it a valuable asset for organizations to leverage modern architectures.

### Motivation for joining Commonhaus

For Debezium to remain the leading open-source platform for Change Data Capture, we must consider long-term goals to enable the project to adapt to a fast-paced, highly innovative landscape, which includes:

* Continue to grow our community
* Increase the adoption and awareness of Debezium
* Embrace transparency in our roadmap and decision-making processes
* Encourage multi-vendor participation and execution

Joining CommonHaus will enable us to achieve these goals organically and benefit everyone involved.

### Things that must be done (bootstrap phase)

- [X] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting).
- [X] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [x] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by Naros** [comment]
[Debezium bootstrapping agreement, 2024-10-30](https://github.com/commonhaus/foundation/pull/211)

**Comment by gunnarmorling** [comment]
Very cool to see this initiative, kudos to everyone involved!

**Comment by JooHyukKim** [comment]
Welcome Debezium team!

**Comment by gavinking** [comment]
Awesome!

**Comment by ebullient** [comment]
🎉 Welcome to Commonhaus!

**Reply by ebullient** [reply]
@Naros .. you have a pending invitation to the organization. Accept that, and we can get the rest started.

**Reply by jpechane** [reply]
@ebullient Hi, I am not sure if Chris managed to do it or not but his is OOO this week - so just FYI.

**Reply by ebullient** [reply]
I can now support second/backup rep as well. Y'all should probably think about that.

**Comment by gavinking** [comment]
Excellent!!

**Comment by gunnarmorling** [comment]
@jpechane, @Naros, do we have a process yet for joining this effort? I.e. what would it take for me to become an official contributor to Debezium under the CommonHaus reign?

**Reply by jpechane** [reply]
@gunnarmorling Not yet (the process), let's discuss this the next week.

**Comment by gunnarmorling** [comment]
SGTM, hit me up any time.

**Comment by criccomini** [comment]
If y'all are lookin' for a GOVERNANCE.md, SlateDB picked up Vespa's (https://github.com/vespa-engine/vespa/blob/master/GOVERNANCE.md) and modified it to fit our needs. I found it to be a good starting point.
